package.json

```
@@ -19,7 +19,11 @@
     "size": "size-limit",
     "analyze": "size-limit --why"
   },
-  "peerDependencies": {},
+  "peerDependencies": {
+    "react": ">=16",
```

```suggestion
    "react": "^16.8.0 || ^17.0.0",
```
hooks were introduced in 16.8